cpp/docs/BENCHMARKING.md

```diff
@@ -39,7 +39,8 @@ performance in repeated iterations.
 
 For generating benchmark input data, helper functions are available at [cpp/benchmarks/common/generate_input.hpp](/cpp/benchmarks/common/generate_input.hpp). The input data generation happens on device, in contrast to any `column_wrapper` where data generation happens on the host.
 * `create_sequence_table` can generate sequence columns starting with value 0 in first row and increasing by 1 in subsequent rows.
-* `create_random_table` can generate a table filled with random data. The random data parameters are configurable.
+* `create_random_column` can generate a column filled with random data. The random data parameters are configurable.
+* `create_random_table` can generate a table of columns filled with random data. Equivalent to multiple calls to `create_random_column`.
```

Comment:
Not really though. Calling `create_random_table` for 2 100 row columns would produce a different second table to the second call to `create_random_column`. I don't know if this nit is with picking though.

Reply:
I wrote it like this mostly to avoid repeating the sentence from the comment above :)
Looks like copy-paste is a better option here :D